Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -8381,6 +8397,57 @@ def domain_command():
         })
 
 
+def ip_command(ips):
+    try:
+        from urllib2 import build_opener, ProxyHandler
+        from ipwhois import IPWhois
+    except ImportError as e:
+        return_error("The Docker needs to be updated to use an IP command")
+
+    results = []
+    for ip in argToList(ips):
+        if demisto.params().get('proxy'):
+            proxies = assign_params(http=handle_proxy().get('http'), https=handle_proxy().get('https'))
+            handler = ProxyHandler(proxies)
+            opener = build_opener(handler)
+            ip_obj = IPWhois(ip, proxy_opener=opener)
+        else:
+            ip_obj = IPWhois(ip)
```

Comment:
Let's put this in a dedicated function
```suggestion
        proxy_opener = None
        if demisto.params().get('proxy'):
            proxies = assign_params(http=handle_proxy().get('http'), https=handle_proxy().get('https'))
            handler = ProxyHandler(proxies)
            opener = build_opener(handler)
            proxy_opener=opener
        ip_obj = IPWhois(ip, proxy_opener)
```